### DISCUSSION TITLE
Demons of Asteborg

### DISCUSSION THREAD

**Discussion by Ruggerio150** [sanni/cartreader#323] [Need help]
Hey. Is it possible to Save and Restore Demons of Asteborg Savegames. I am on Version 6.4 i think. Can the new Firmware read this Cartridge?

**Comment by sanni** [comment]
I have not tried any Mega Drive bootlegs/repros myself yet but as it looks right now they will not work without adjusting the code. Applies to both Demons of Asteborg and also Metal Dragon.
My guess would be that changing the readWord_MD() function so that it resembles the readFlash_MD() function could resolve the issue.

**Comment by Ruggerio150** [comment]
Ah ok. Can i try it on my own? I think the savegame is to Big. On the MISTERFPGA it saves and the file ist 128kb.
